domains/freeutka.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "freeutka",
+    "email": "desingfreeutka@gmail.com"
+  },
+  "record": {
+    "CNAME": "freeutka-cmd.github.io"
```

Comment:
Domain redirects to the is-a.dev domain, please disconnect it from the website so that we can see your website